### ISSUE TITLE
eps and momentum parameters

### ISSUE THREAD

**Issue by gabrer** [jihunchoi/recurrent-batch-normalization-pytorch#11]
The initialization function of the **SeparatedBatchNorm1d** module has two arguments `eps` and `momentum`. 

```
def __init__(self, num_features, max_length, eps=1e-5, momentum=0.1,
                 affine=True):
```

Is this `momentum` the same we use in the in our optimization algorithm (e.g. SGD), or it's an additional momentum just for the batch normalization process?

I couldn't find any mention about this in the original paper.

**Comment by cloudygoose** [comment]
@gabrer 
You can find info in the document of batch_norm
http://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm1d

**Comment by gabrer** [comment]
Thank you! I have already found it! :)
